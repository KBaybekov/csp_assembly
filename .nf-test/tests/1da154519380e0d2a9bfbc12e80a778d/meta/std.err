[0;31mThe following invalid input values have been detected:

* --input (/raid/kbajbekov/common_share/github/csp_assembly/tests/test_samplesheet.csv): Validation of file failed:
	-> Entry 1: Error for field 'short_fastq_1' (${projectDir}/tests/test_data/illumina/GCF_000012125.1_cov_30_1.fq.gz): the file or directory '${projectDir}/tests/test_data/illumina/GCF_000012125.1_cov_30_1.fq.gz' does not exist (FastQ file for reads 1 must be provided, cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz')
	-> Entry 2: Error for field 'short_fastq_1' (tests/test_data/illumina/GCF_000012125.1_cov_30_1.fq.gz): the file or directory 'tests/test_data/illumina/GCF_000012125.1_cov_30_1.fq.gz' does not exist (FastQ file for reads 1 must be provided, cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz')
	-> Entry 3: Error for field 'short_fastq_2' (tests/test_data/illumina/GCF_000012125.1_cov_30_2.fq.gz): the file or directory 'tests/test_data/illumina/GCF_000012125.1_cov_30_2.fq.gz' does not exist (FastQ file for reads 2 cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz')
	-> Entry 3: Error for field 'short_fastq_1' (tests/test_data/illumina/GCF_000012125.1_cov_30_1.fq.gz): the file or directory 'tests/test_data/illumina/GCF_000012125.1_cov_30_1.fq.gz' does not exist (FastQ file for reads 1 must be provided, cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz')
	-> Entry 4: Error for field 'short_fastq_2' (tests/test_data/illumina/GCF_000012125.1_cov_30_2.fq.gz): the file or directory 'tests/test_data/illumina/GCF_000012125.1_cov_30_2.fq.gz' does not exist (FastQ file for reads 2 cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz')
	-> Entry 4: Error for field 'short_fastq_1' (tests/test_data/illumina/GCF_000012125.1_cov_30_1.fq.gz): the file or directory 'tests/test_data/illumina/GCF_000012125.1_cov_30_1.fq.gz' does not exist (FastQ file for reads 1 must be provided, cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz')
	-> Entry 5: Error for field 'short_fastq_2' (tests/test_data/illumina/GCF_000012125.1_cov_30_2.fq.gz): the file or directory 'tests/test_data/illumina/GCF_000012125.1_cov_30_2.fq.gz' does not exist (FastQ file for reads 2 cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz')
	-> Entry 5: Error for field 'short_fastq_1' (tests/test_data/illumina/GCF_000012125.1_cov_30_1.fq.gz): the file or directory 'tests/test_data/illumina/GCF_000012125.1_cov_30_1.fq.gz' does not exist (FastQ file for reads 1 must be provided, cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz')
[0m
 -- Check script '/raid/kbajbekov/common_share/github/csp_assembly/subworkflows/nf-core/utils_nfschema_plugin/main.nf' at line: 39 or see '/raid/kbajbekov/common_share/github/csp_assembly/.nf-test/tests/1da154519380e0d2a9bfbc12e80a778d/meta/nextflow.log' file for more details
