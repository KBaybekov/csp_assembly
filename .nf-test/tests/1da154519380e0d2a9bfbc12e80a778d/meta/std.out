ERROR ~ Validation of pipeline parameters failed!

 -- Check '/raid/kbajbekov/common_share/github/csp_assembly/.nf-test/tests/1da154519380e0d2a9bfbc12e80a778d/meta/nextflow.log' file for details
